src/utilities/new_versions.jl

```diff
@@ -280,24 +295,69 @@ function make_pr_for_new_version(
             if commit_was_success
                 @info("Commit was a success")
                 api_retry() do
-                    @mock git_push("origin", new_branch_name; force=true, env=env)
+                    @mock git_push(
+                        "origin", new_branch_name, pkey_filename; force=true, env=env
+                    )
                 end
 
-                create_new_pull_request(
+                new_pr, _ = create_new_pull_request(
                     forge,
                     repo,
                     new_branch_name,
                     master_branch_name,
                     new_pr_title,
                     new_pr_body,
                 )
+
+                cc_user && cc_trigger_user(forge, repo, new_pr; env=env)
+                unsub_from_prs && unsub_from_pr(forge, new_pr)
             end
         end
     end
 
     return nothing
 end
 
+function cc_trigger_user(
+    api::GitHub.GitHubAPI, repo::GitHub.Repo, pr::GitHub.PullRequest;
+    env=ENV,
+)
+    username = env["GITHUB_ACTOR"]
+    body = "cc @$username"
+
+    return @mock GitForge.create_pull_request_comment(
+        api,
+        repo.owner.login,
+        repo.name,
+        pr.id;
+        body=body,
+    )
+end
+
+function cc_trigger_user(
+    api::GitLab.GitLabAPI, repo::GitLab.Project, pr::GitLab.MergeRequest;
+    env=ENV,
+)
+    username = env["GITLAB_USER_LOGIN"]
+    body = "cc @$username"
+
+    return @mock GitForge.create_pull_request_comment(
```

Comment:
why are these different?
Point of GitForge is that they should have common API

Reply:
Unfortunately gitforge isn't perfect in this regard. Some API calls for gitlab specifically need an owner and repo as strings, while in gitlab require the project id. This causes some situations where you need to separate how github and gitlab do calls. This also appears in the fact that the naming of classes/variables/kwargs is different for the "same" command on gitlab/hub

Reply:
We should fix GitForge to take a higher level object as a argument, that contains as needed both owner/repo, and project id.

Reply:
This seems very out of scope for right now. I think it would be best to make an issue, but this seems like it would be a big change to overhaul gitforge to properly be more generic

Reply:
that being said, I think it would be nice to continue to show love to gitforge and contribute to make it better